views/pricing.erb

```diff
@@ -12,16 +12,152 @@
 			<%== erb :'partials/_subnav', :locals => {active: 'pricing'} %>
 		</div>
 
-
 		<div class="column-two-thirds">
-			<h1 class="heading-xlarge">Pricing</h1>
-			<p>
-				You pay for what you use, like hosting, compute resource and backing services such as a database.
-			</p>
-			<p>The overall cost depends on factors such as the number of environments you deploy to, the size and number of your applications, the size of your database services, and whether they need to have high availability.</p>
+      <h1 class="heading-xlarge">Pricing</h1>
```

Comment:
Looks like tabs for indentation in this file...

Reply:
correct